plugins/test/unit/plugins/db/test_models.py

```diff
@@ -399,6 +400,9 @@ def test_update_needed_different_supported_date_formats(self):
         uploaded_erratum.updated = '2016-01-01 00:00:00'
         ret = existing_erratum.update_needed(uploaded_erratum)
         self.assertTrue(ret)
+        uploaded_erratum.updated = '2016-01-01 00:00'
```

Comment:
+1 I think it's fine to add it here, since the test is supposed to check the handling of all the supported date formats.